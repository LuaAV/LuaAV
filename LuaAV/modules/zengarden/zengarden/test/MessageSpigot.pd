#N canvas 1095 634 450 300 10;
#X obj 107 168 spigot;
#X obj 107 5 loadbang;
#X obj 176 6 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 107 214 print;
#X msg 164 132 0;
#X msg 195 132 1;
#X msg 228 132 9923;
#X obj 107 43 t b b b b b b b;
#X msg 97 92 hello;
#X msg 54 92 bang;
#X msg 14 93 0;
#X msg 147 91 -4.2218;
#X connect 0 0 3 0;
#X connect 1 0 7 0;
#X connect 2 0 7 0;
#X connect 4 0 0 1;
#X connect 5 0 0 1;
#X connect 6 0 0 1;
#X connect 7 0 10 0;
#X connect 7 1 6 0;
#X connect 7 2 9 0;
#X connect 7 3 4 0;
#X connect 7 4 8 0;
#X connect 7 5 5 0;
#X connect 7 6 11 0;
#X connect 8 0 0 0;
#X connect 9 0 0 0;
#X connect 10 0 0 0;
#X connect 11 0 0 0;
