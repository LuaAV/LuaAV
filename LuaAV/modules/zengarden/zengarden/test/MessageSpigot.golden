[@ 0.000ms] print: hello
[@ 0.000ms] print: 0